### PR TITLE
images: Move rhel-10-0 from beta to final

### PR DESCRIPTION
**Pull request by martinpitt**
* [x] image-refresh rhel-10-0
 * [x] https://github.com/cockpit-project/cockpit/pull/21264
 * [ ] https://github.com/cockpit-project/cockpit-machines/pull/1909

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh rhel-10-0 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-10-0-20241113-165626

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-rhel-10-0-0520313e-20241113-164028/log.html

**Comment by martinpitt** [comment]
This is quite a large jump, I expect the same pcp regressions / FIPS changes as in #7091. Let's see!

**Comment by martinpitt** [comment]
Kernel oops with FIPS: https://issues.redhat.com/browse/RHEL-65498 (could require a naughty). Fortunately this doesn't yet break centos-10 (#7091), so testing my changes on that image instead.

**Comment by martinpitt** [comment]
FIPS fixed in https://github.com/cockpit-project/cockpit/pull/21264  (blocking). Tests running there.

**Comment by cockpituous** [comment]
image-refresh rhel-10-0 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-10-0-20241114-110909

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-rhel-10-0-e02a9c5b-20241114-105121/log.html

File: images/scripts/rhel.setup [hunk 1]
```diff
@@ -298,6 +297,12 @@ stratisd
...
+fi
```
Comment (jelly) [inline images/scripts/rhel.setup:304]: :see_no_evil:  feels appropriate here :)

**Review by jelly — APPROVED** [review]

File: images/scripts/rhel.setup (hunk 1 above)
Comment (martinpitt) [reply images/scripts/rhel.setup:304]: Look behind! A three-headed monkey! :monkey_face: :monkey_face: :monkey_face:

**Comment by martinpitt** [comment]
Meh, [testAddDiskNFS](https://cockpit-logs.us-east-1.linodeobjects.com/pull-7095-582eaaf2-20241115-115858-rhel-10-0-cockpit-project-cockpit-machines/log.html#26-2) puts up a good fight, that smells like more NFS fallout. Investigating..

**Review by mvollmer — APPROVED** [review]
Not pretty, but "ok". :-)

**Comment by martinpitt** [comment]
Fun, that test takes very long, due to some SELinux problem, and then unceremoniously reboots the VM without a further log entry. That smells an awful lot like https://github.com/cockpit-project/cockpit-machines/issues/1900 ? 

This command:

    virsh vol-create-as --pool nfs-pool --name nfs-volume-0 --capacity 1M --format qcow2

also takes suspiciously long, like half a minute.

It passes with `setenforce 0`. Checking this more closely..

**Comment by martinpitt** [comment]
https://github.com/cockpit-project/cockpit-machines/pull/1909 should help, WFM locally.

**Comment by jelly** [comment]
Uhhh!

**Comment by jelly** [comment]
I'd like machines to go green here do, so needs a retry.

**Pull request merged**